packages/usa-identifier/src/styles/_usa-identifier.scss

```diff
@@ -142,7 +142,8 @@ $identifier-links-gap: 4 !default;
   }
 }
 
-.usa-identifier__required-link {
+.usa-identifier__required-link,
+.usa-identifier__required-link.usa-link {
```

Comment:
Prevent `usa-link` from overriding Identifier styles, regardless of where the usa-link styles fall in the cascade